app/views/providers/means/add_other_state_benefits/show.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= form_with(
+      url: providers_legal_aid_application_means_add_other_state_benefits_path,
+      model: @form,
+      method: :patch,
+      local: true,
+    ) do |form| %>
+  <%= page_template page_title: t(".page_title"), form:, template: :basic do %>
+    <span class="govuk-caption-l"><%= t(".caption") %></span>
+    <h1 class="govuk-heading-xl"><%= t(".existing", count: pluralize(@transactions.count, "benefit").to_s) %></h1>
+      <%= govuk_summary_list(classes: "has_other_dependants") do |summary_list| %>
```

Comment:
Should this say 'has other benefits' not 'has other dependants' ?

Reply:
What's the front end perspective on this?
It's a css class for summary lists that is _now_ used for multiple things
Should we 
a) rename it to `has_other_list`, or; 
b) duplicate it for each type?

My heart says a) but happy to discuss :)